Rakefile

```diff
@@ -33,3 +33,15 @@ task :cleanup_old_stories, [:number_of_days] => :environment do |_t, args|
   args.with_defaults(number_of_days: 30)
   RemoveOldStories.call(args[:number_of_days].to_i)
 end
+
+namespace :env do
+  desc "Generate Postgres env file"
+  task :generate_postgres do
+    File.write("./.postgres.env", `erb ./docker/.postgres.env.erb`)
```

Comment:
Thinking about these files, do we really need them to be separate? What if we had a single `.env.production.local` file that includes both of these?

Side note, we're using the dotenv gem. I'm not sure how that interplays with any builtin `.env` support Docker has, but they have a [list of recommendations](https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use).

Reply:
I'll take a look through those recs.

We should have the env files separate imo because someone might not want to run our provided postgres container but rather their own (with other dbs in it). It also is probably better from a security standpoint, I think?

Reply:
Hmm, I guess I'm not sure how all the pieces will fit together here. I'm not able to think of any security benefit to keeping them separate. Users can edit the file afterwords if they want to point to their own PG installation, right? By default, dotenv [won't override existing env variables](https://github.com/bkeepers/dotenv#why-is-it-not-overriding-existing-env-variables), so if they have it setup somewhere else that should take precedence.

I'm mostly concerned about people having to be aware of multiple `.env` files that might impact their installation. It seems easier to have it all in one place.

Reply:
I guess that's true, as long as we don't have the PG variables required in Dotenv.